samples/MvcApp/Services/CustomClaimsSerializer.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Claims;
+using Microsoft.AspNetCore.SystemWebAdapters.Authentication;
+
+namespace MvcApp.Services
+{
+    internal class CustomClaimsSerializer : IClaimsSerializer
+    {
+        public ClaimsPrincipal Deserialize(Stream responseContent)
+        {
+            throw new NotImplementedException();
```

Comment:
Why not have one serializer that shares code? That way it's clear that they serialize/deserialize to the same structure

Reply:
Hm... I see what you're saying but it's not obvious where that shared source would live. Some options -

1. We could put it in ClassLibrary, but that project targets .NET Standard and the `IClaimsSerializer` interface is in the FrameworkServices/CoreServices packages which don't target .NET Standard. We could move `IClaimsSerializer` to Microsoft.AspNEtCore.SystemWebAdapters which does target .NET Standard, but I feel like the interface logically fits in the service-specific packages more than the adapters one.
2. We could have a shared source file that is included in both the MvcApp and MvcCoreApp sample projects. This is probably what I would do in a real-world scenario, but we don't currently have any shared source between those two sample projects and it's unclear to me if it's worth complicating our samples folder with an extra 'shared' folder or linking the file from one project to another for this case.

I could go either way on this one. I think the way it is in the PR now is ok, but if you prefer option 2 above that also seems perfectly reasonable and I'd be fine adding an MvcShared folder for the serializer class.

Reply:
This may be a good reason to have another package of `.Services`. The same issue will be hit with the key serialization. These are logically the same type, and we should probably do something so that they *are* the same type.

Reply:
Yea, if there was a `.Services` package, then the netstandard library used by the sample could depend on it and have the serializer implementation live there.